### PR TITLE
Added todo code for review

### PR DESCRIPTION
**Pull request by audukehinde**
- Added todo list code for review

### CODE REVIEW

File: src/displayTodo.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+    todoList.appendChild(todoItem);
+
+    if (todo.done) {
+      todoItem.classList.add('done');
+      deleteButton.style.display = 'block';
+    }
+
+    input.addEventListener('click', (e) => {
+      todo.done = e.target.checked;
```
Comment (lenkon) [inline src/displayTodo.js:49]: Hi @audukehinde, it is better to give meaningful self-explanatory names for variables according to JavaScript clean code best practices. Please change it wherever necessary.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import './style.css';
+import getFromLocalSTorage from './myLocalStorage.js';
+import displayTodo from './displayTodo.js';
+
+window.addEventListener('load', () => {
```
Comment (lenkon) [inline src/index.js:6]: Hi @audukehinde, this function is performing many actions. According to JavaScript clean code best practices a function should do one thing. Please change it into multiple smaller functions.

**Review by lenkon — APPROVED** [review]
Hi @audukehinde ,

## Highlights!
- No linter errors ✔️ 
- Good code organization and working app ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Aspects to improve ♻️

- [x] Check the comments under the review.

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/displayTodo.js (hunk 1 above)
Comment (audukehinde) [reply src/displayTodo.js:49]: Oh okay. I'll put that to heart. Thanks

File: src/index.js (hunk 2 above)
Comment (audukehinde) [reply src/index.js:6]: Okay, I'll do that. Thanks

**Review by audukehinde — COMMENTED** [review]
Function performing one task

**Pull request merged**